lib/lambda.js

```diff
@@ -453,10 +453,30 @@ function isLambdaExecutionEnvironment () {
   return !!process.env.AWS_LAMBDA_FUNCTION_NAME
 }
 
+function getLambdaHandler (env) {
+  if (!isLambdaExecutionEnvironment() || !env._HANDLER || !env.LAMBDA_TASK_ROOT) {
+    return
+  }
+
+  const [handlerModule, handlerFunction, extra] = env._HANDLER.split('.')
```

Comment:
Lambda allows "myModules.some.nestedpathto.myHandler". https://github.com/aws/aws-lambda-nodejs-runtime-interface-client/blob/c31c41ffe5f2f03ae9e8589b96f3b005e2bb8a4a/src/utils/UserFunction.ts#L39-L58

Reply:
So they do -- tests and logic added to handle this a12ed5ab9dbdf21f3f98100bf98f445d08b7c260

Reply:
Unresolved this -- while testing it I'd misunderstood the assignment.

The _correct_ semantics of `myModules.some.nestedpathto.myHandler` are a _module_ named `myModules` that exports a handler that looks something like this

```
module.exports = {
  some:{
    nestedpathto: {
        myHandler:function() { } 
    }
  }
}
```

It does _not_ allow a nested file path. I intend to re-implement using this new information.

Reply:
OK, got this one working. Wrapping's been manually tested with handlers  named `index.handler` and `science.testmore.handler` and data successfully reported in.